EpikCord/flags.py

```diff
@@ -0,0 +1,131 @@
+from __future__ import annotations
+
+from typing import Any, Dict, List
+
+
+class Flag:
+    class_flags: Dict[str, int]
+
+    def __init_subclass__(cls) -> None:
+        cls.class_flags = {k: v for k, v in cls.__dict__.items() if isinstance(v, int)}
+
+    def __init__(self, value: int = 0, **kwargs):
+        self.value = value
+        self.turned_on: List[str] = [k for k, a in kwargs.items() if a]
+
+        for k, v in self.class_flags.items():
+            if v & value and k not in self.turned_on:
+                self.turned_on.append(k)
+
+        self.calculate_from_turned()
+
+    def calculate_from_turned(self):
+        value = 0
+        for key, flag in self.class_flags.items():
+            if key in self.class_flags:
+                value |= flag
+        self.value = value
+
+    def __getattribute__(self, __name: str) -> Any:
```

Comment:
`Any` annotation makes this horrible to work with, especially with dynamic methods like get attribute